From cda63355208cf9e1c702b2857f911c503087ed94 Mon Sep 17 00:00:00 2001
From: Dominik Csapak <dominik.csapak@gmail.com>
Date: Sun, 2 Aug 2020 10:05:20 +0200
Subject: [PATCH 2/2] remove apt dependency

not needed for the client, and would depend on libapt-pkg-dev

Signed-off-by: Dominik Csapak <dominik.csapak@gmail.com>
Signed-off-by: Thomas Lamprecht <thomas@lamprecht.org>
---
 Cargo.toml           |   1 -
 src/tools.rs         |   1 -
 src/api2/node.rs     |   2 -
 3 files changed, 4 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 99b55558..720b8f52 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -14,7 +14,6 @@ name = "proxmox_backup"
 path = "src/lib.rs"
 
 [dependencies]
-apt-pkg-native = "0.3.1" # custom patched version
 base64 = "0.12"
 bitflags = "1.2.1"
 bytes = "0.5"
diff --git a/src/tools.rs b/src/tools.rs
index e83764e4..5ef3a241 100644
--- a/src/tools.rs
+++ b/src/tools.rs
@@ -19,7 +19,6 @@ use proxmox::tools::vec;
 pub use proxmox::tools::fd::Fd;
 
 pub mod acl;
-pub mod apt;
 pub mod async_io;
 pub mod borrow;
 pub mod cert;
diff --git a/src/api2/node.rs b/src/api2/node.rs
index a19bea7e..346bb04f 100644
--- a/src/api2/node.rs
+++ b/src/api2/node.rs
@@ -24,7 +24,6 @@ use crate::server::WorkerTask;
 use crate::tools;
 use crate::tools::ticket::{self, Empty, Ticket};
 
-pub mod apt;
 pub mod disks;
 pub mod dns;
 pub mod network;
@@ -306,7 +305,6 @@ fn upgrade_to_websocket(
 }
 
 pub const SUBDIRS: SubdirMap = &[
-    ("apt", &apt::ROUTER),
     ("disks", &disks::ROUTER),
     ("dns", &dns::ROUTER),
     ("journal", &journal::ROUTER),
-- 
2.28.0
